hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/service/TestDirectoryDeletingService.java

```diff
@@ -76,6 +87,7 @@ private OzoneConfiguration createConfAndInitValues() throws IOException {
     ServerUtils.setOzoneMetaDirPath(conf, newFolder.toString());
     conf.setTimeDuration(OZONE_DIR_DELETING_SERVICE_INTERVAL, 3000,
         TimeUnit.MILLISECONDS);
+    conf.setInt(OZONE_THREAD_NUMBER_DIR_DELETION, threadCount);
```

Comment:
Unrelated: ozone.thread.number.dir.deletion should be renamed to ozone.om.thread.number.dir.deletion.
Along with a bunch other configuration property names in OMConfigKeys